src/config.rs

```diff
@@ -343,6 +378,19 @@ impl Config {
     /// # Returns
     ///
     /// The loaded global config
+    /// Validate the config file.
```

Comment:
The doc is misaligned

Reply:
Dammit thanks for noticing!
Was a pretty big conflict, and the magic wand didn't do its magic properly